### ISSUE TITLE
add kernel version to telemetry traces.

### ISSUE THREAD

**Issue by jimassa** [microsoft/retina#1071]
heartbeat already comes with kernel version attribute; https://github.com/microsoft/retina/blob/5ff1567e8a988209494b0cddd34f3939ac51e53c/pkg/telemetry/telemetry.go#L170

We would like to add it to traces as well in order to directly pinpoint what kernel version is producing a specific log.

**Comment by Anirudh2112** [comment]
Hi @jimassa, I would like to contribute to this

**Comment by Anirudh2112** [comment]
Hello @jimassa I have made the necessary changes and committed a PR. Kindly review it. Thank you.

**Comment by matmerr** [comment]
isn't it possible to correlate pod name with kernel version in heartbeat?

**Comment by Anirudh2112** [comment]
Thank you for the suggestion! I will add the pod name to the heartbeat properties to enable correlation with the kernel version. I'll update the PR shortly. Please let me know if there are any other changes/features to be implemented.

**Comment by matmerr** [comment]
podname should already be a part of telemetry properties, so it should be all good

https://github.com/microsoft/retina/blob/8963dd561d4d5b52c1f711dcbc13fe9f616bd360/pkg/telemetry/telemetry.go#L155

**Comment by Anirudh2112** [comment]
Yes, In the TrackEvent method, t.properties is merged with the provided event properties, so podname will automatically be included in the heartbeat telemetry.
